### ISSUE TITLE
[Accessibility][Screen reader]: Narrator announces same position number for all facepile controls. (Bug 6)

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10860]
Tracking all of the issues brought up with the 1/4 accessibility pass on Facepile control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bfd43fcc5-515d-47d8-946d-0d4862a56278%7D&action=default&uid=%7BFD43FCC5-515D-47D8-946D-0D4862A56278%7D&ListItemId=1647&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 

Breakout of issue #7534

**Comment by aneeshack4** [comment]
This no longer repros - Narrator doesn't read out "1 of 1" as described in this [bug report](https://microsoft-my.sharepoint.com/personal/v-amaver_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fv%2Damaver%5Fmicrosoft%5Fcom%2FDocuments%2FFabric%20Control%2FP1%20Fabric%20Control%2FFacepile%2FBug%5F06%2Emp4&parent=%2Fpersonal%2Fv%2Damaver%5Fmicrosoft%5Fcom%2FDocuments%2FFabric%20Control%2FP1%20Fabric%20Control%2FFacepile&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOnY6L3Avdi1hbWF2ZXIvRVVuZGY5eFNaekpLbnBwMllKRHB5amtCZXRlTTVwVWtUVTBUdzExOERqdTZkQT9ydGltZT1CRkswaDMyQzEwZw
) on the Facepiles. But perhaps it should? I think the correct behavior is for it read out something like "Annie Lindqvist, 1 of 4" tagging component owners @mtennoe @Markionium for additional input.

**Comment by mtennoe** [comment]
Should the facepile items be focusable by default btw? I would think only if they are interactable?
I would prefer it to read "X of N", as that provides more context to the user. Would have to mean the consumer passes in a localized format string though, which complicates things a bit. I'll let @Markionium comment as well

**Comment by aneeshack4** [comment]
The team that owns Facepile is tracking this separately and will get a fix for it out soon. For now I'll assign this to @mtennoe and @Markionium so they can update these issues when those fixes are done.
